_data/changelog.yml

```diff
@@ -1,3 +1,8 @@
+- date: February 5, 2020
+  title: "TBD"
+  description: "TBD"
```

Comment:
```suggestion
  description: "We've added terms for two new services: CIS compliance and professional services. We've also removed Houston for now since we are not currently accepting new Houston customers. Finally, we've cleaned up a few minor legal positions around our billing practices, termination, and handling your feedback."
```